{
	"id": "SUSE-SU-2021:2253-1",
	"modified": "2021-07-06T07:03:40Z",
	"published": "2021-07-06T07:03:40Z",
	"related": [
		"CVE-2020-13757"
	],
	"summary": "Security update for python-rsa",
	"details": "This update for python-rsa fixes the following issues:\n\n- CVE-2020-13757: Proper handling of leading '\\0' bytes during decryption of ciphertext (bsc#1172389)\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud 9",
				"name": "python-rsa",
				"purl": "purl:rpm/suse/python-rsa&distro=SUSE%20OpenStack%20Cloud%209"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.4.2-4.4.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-rsa": "3.4.2-4.4.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:OpenStack Cloud Crowbar 9",
				"name": "python-rsa",
				"purl": "purl:rpm/suse/python-rsa&distro=SUSE%20OpenStack%20Cloud%20Crowbar%209"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.4.2-4.4.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-rsa": "3.4.2-4.4.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2021/suse-su-20212253-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1172389"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2020-13757"
		}
	]
}
